Code/Framework/AzToolsFramework/Tests/Prefab/PrefabTestFixture.cpp

```diff
@@ -76,6 +97,42 @@ namespace UnitTest
         return newEntity;
     }
 
+    AZ::EntityId PrefabTestFixture::CreateEntityWithPrefab(const char* entityName, AZ::EntityId parentId)
+    {
+        auto createResult = m_prefabPublicInterface->CreateEntity(parentId, AZ::Vector3());
+        AZ_Assert(createResult.IsSuccess(), "Failed to create entity: %s", createResult.GetError().c_str());
+        AZ::EntityId entityId = createResult.GetValue();
+
+        AZ::Entity* entity = nullptr;
+        AZ::ComponentApplicationBus::BroadcastResult(entity, &AZ::ComponentApplicationRequests::FindEntity, entityId);
+
+        entity->Deactivate();
+
+        entity->SetName(entityName);
```

Comment:
nit: As `SetName` takes an `AZStd::string` the `entityName` parameter might as well be an `AZStd::string` as opposed to a `const char*` (you can then lose the calls to `.c_str()`) or you could update it to `string_view` which is fine in this case as we're using it as a `string` and not passing it to an API expecting a null terminated string

Reply:
fixed